### PR TITLE
Unspecialize test hash sizes

### PR DESCRIPTION
**Pull request by lehins**
Avoid mock crypto when possible in favor of StandardCrypto.

Stop using small blake hashes for testing.

### CODE REVIEW

File: eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs [hunk 1]
```diff
@@ -72,9 +72,8 @@ import NoThunks.Class (NoThunks (..), allNoThunks)
 -- ===============================================================
 
 -- | The result of reward calculation is a pair of aggregate Maps.
-data RewardAns c
-  = RewardAns
-      !(Map (Credential 'Staking c) (Reward c))
+newtype RewardAns c
+  = RewardAns (Map (Credential 'Staking c) (Reward c))
```
Comment (JaredCorduan) [inline eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs:76]: why did you remove the strictness annotation?

**Review by JaredCorduan — APPROVED** [review]
If you can make CI happy I think it is good to go. You did a lot of clean-up, much appreciated!

File: eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs (hunk 1 above)
Comment (lehins) [reply eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs:76]: Because I changed it to a `newtype` :)

**Comment by lehins** [comment]
@JaredCorduan Awesome. Thank you for the review.

I am still battling with one test case, so it is almost done.

File: eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs (hunk 1 above)
Comment (JaredCorduan) [reply eras/shelley/impl/src/Cardano/Ledger/Shelley/RewardUpdate.hs:76]: ah! :)
